### PR TITLE
Display representative media for office documents

### LINKED ISSUE #261: Office documents don't display representative image on work show page.
wustl 076 - representative image not displaying for documents

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes #261

### CODE REVIEW

File: spec/presenters/curation_concerns/generic_file_presenter_spec.rb [hunk 1]
```diff
@@ -26,4 +27,9 @@
     subject { presenter.to_partial_path }
     it { is_expected.to eq 'generic_files/generic_file' }
   end
+
+  describe "office_document?" do
+    subject { presenter.office_document? }
+    it { is_expected.to be false }
+  end
 end
```
Comment (cam156) [inline spec/presenters/curation_concerns/generic_file_presenter_spec.rb]: @jcoyne Is there a test for when office_document? is true?

Comment (jcoyne) [inline spec/presenters/curation_concerns/generic_file_presenter_spec.rb]: This test just shows its proxies to the Solr doc. The test for MimeTypes is in Hydra-Works

Comment (cam156) [inline spec/presenters/curation_concerns/generic_file_presenter_spec.rb]: :ok_hand:

**Comment by cam156** [comment]
@jcoyne unfortunately it looks like this PR need a rebase

**Pull request merged**